libraries/ui-library/src/components/six-error/six-error.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * @since 4.0
+ * @status beta
+ *
+ * @slot default - The error's message content
+ */
+
+@Component({
+  tag: 'six-error',
+  styleUrl: 'six-error.scss',
```

Comment:
Most other components set `shadow:true`. Any particular reason it is not set here?